### PR TITLE
Batch 7

### PR DESCRIPTION
**Pull request by SleepingSoul**

### CODE REVIEW

File: Tihran_Katolikian/7/Character.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
+from UpdateList import UpdateList
+import DungeonGameSaveLoad
+from enum import Enum
+from LoggerDecorator import logger_decorator
+import logging
+from DungeonMap import DungeonCell
+import DungeonGameConfig
+
+
+class PlayerCommand(Enum):
```
Comment (oleksandr-medviediev) [inline Tihran_Katolikian/7/Character.py]: Redundant class usage.

File: Tihran_Katolikian/7/DungeonGame.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+from Character import Character
+from LoggerDecorator import logger_decorator
+import logging
+from DungeonMap import DungeonMap
+import DungeonGameConfig
+import DungeonGameSaveLoad
+
+
+class GameStartMode(Enum):
+    NEW_GAME = 0
```
Comment (oleksandr-medviediev) [inline Tihran_Katolikian/7/DungeonGame.py]: Redundant class usage.

File: Tihran_Katolikian/7/DungeonGame.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    LOAD_GAME = 1
+
+
+class DungeonGameState(Enum):
+    GAME_MENU = 0
```
Comment (oleksandr-medviediev) [inline Tihran_Katolikian/7/DungeonGame.py]: Redundant class usage.

**Comment by SleepingSoul** [comment]
Looks like I named all "private" field with two underscores: __ when I should have named it with one underscore _. Probably it is because I was reading the presentation inattentively ;) I will fix it as soon as possible.

**Pull request merged**